<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCadastroInstrumentos</class>
 <widget class="QWidget" name="frmCadastroInstrumentos">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>632</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>632</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>632</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Instrumentos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frmCadastroInstrumentos
{
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 94, 255), stop:1 rgba(0, 85, 255, 255));
   font: &quot;URW Gothic L&quot;;
}

#gpbInstrumentos
{
	background-color: rgba(0, 0, 0, 50);
}

#btnCancelar
{
	background-color: rgb(106, 0, 0);
}

#btnLimpar
{	
	background-color: rgb(89, 89, 89);
}

#btnConcluir
{
	background-color: rgb(0, 85, 0);
}</string>
  </property>
  <widget class="QLabel" name="lblCadastroInstrumentos">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>391</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>391</width>
     <height>21</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CADASTRO DE INSTRUMENTOS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConcluir">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Concluir</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimpar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QFrame" name="gpbInstrumentos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>471</width>
     <height>511</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>471</width>
     <height>511</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel, .QPlainTextEdit, .QPushButton, .QComboBox, .QRadioButton, .QLineEdit
{
	font: 12pt &quot;URW Gothic L&quot;;;
}

QPlainTextEdit, .QLineEdit
{
	background-color: rgb(0, 0, 0, 100);
}

QComboBox
{	
	background-color: rgb(45, 45, 45);
}

.QPushButton
{	
	background-color: rgb(65, 65, 65, 150);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblCamposObrigatorios">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>21</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Campos obrigatórios*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPreco">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string>Preço</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbInstrumento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>311</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>32</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Instrumento*</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblEstado">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>61</width>
      <height>21</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Estado*</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtUsado">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>71</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>71</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Usado</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="rtxInformacoes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>451</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>451</width>
      <height>181</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>451</width>
      <height>181</height>
     </size>
    </property>
    <property name="plainText">
     <string>Informações adicionais</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImagem">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>71</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>71</width>
      <height>21</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Imagem</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbTipo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>311</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>32</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Tipo*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cordas</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Percussão</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sopro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Teclas</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtQuantidade">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>451</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>451</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string>Quantidade</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ptbImagem">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>100</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>71</width>
      <height>121</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>121</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Imagens/Others/Microfone.png) 0 0 0 0 stretch stretch;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtNovo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Novo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMarca">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string>Marca</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>321</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string>Nome*</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCarregar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Carregar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtNome</tabstop>
  <tabstop>cmbTipo</tabstop>
  <tabstop>cmbInstrumento</tabstop>
  <tabstop>rbtUsado</tabstop>
  <tabstop>rbtNovo</tabstop>
  <tabstop>txtMarca</tabstop>
  <tabstop>txtPreco</tabstop>
  <tabstop>txtQuantidade</tabstop>
  <tabstop>rtxInformacoes</tabstop>
  <tabstop>btnCarregar</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>btnLimpar</tabstop>
  <tabstop>btnConcluir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
